howmanypeoplearearound/__main__.py

```diff
@@ -73,27 +95,34 @@ def fileToMacSet(path):
 @click.option('-m', '--manufacturers', default='', help='read list of known manufacturers from file')
 @click.option('--nocorrection', help='do not apply correction', is_flag=True)
 @click.option('--aggbymanufacturer', help='do not store mac addresses in the output file, instead store aggregate count of cellphones by manufacturer.', is_flag=True)
+@click.option('--sheetname', help='name of google sheet to write to.', default='')
+@click.option('--googlecreds', help='if writing to a google sheet, file which contains credentials to write to the sheet.', default='')
```

Comment:
Does it make sense to combine both of these flags into a single flag given that we'll never be able to use one without the other?

Reply:
That's a good question, what do you have in mind? I'm not sure what the single flag would look like